### PR TITLE
Add warnings when the PTS or DTS are received out of order

### PR DESCRIPTION
**Pull request by varsill**

### CODE REVIEW

File: lib/membrane_mp4/muxer/cmaf.ex [hunk 1]
```diff
@@ -125,6 +127,20 @@ defmodule Membrane.MP4.Muxer.CMAF do
   def handle_process(Pad.ref(:input, _id) = pad, sample, ctx, state) do
     use Ratio, comparison: true
 
+    if state.last_pts > sample.pts do
+      Membrane.Logger.warn(
+        "PTS are out of order. Current PTS: #{sample.pts} Previous PTS: #{state.last_pts}"
```
Comment (mat-hek) [inline lib/membrane_mp4/muxer/cmaf.ex:132]: I'd point out that this probably indicates B-frames which are not supported

File: lib/membrane_mp4/muxer/cmaf.ex [hunk 2]
```diff
@@ -125,6 +127,20 @@ defmodule Membrane.MP4.Muxer.CMAF do
...
+      )
+    end
+
+    if state.last_dts > sample.dts do
+      Membrane.Logger.warn(
+        "DTS are out of order. Current DTS: #{sample.dts} Previous DTS: #{state.last_dts}"
+      )
+    end
+
+    state = %{state | last_pts: sample.pts, last_dts: sample.dts}
```
Comment (mat-hek) [inline lib/membrane_mp4/muxer/cmaf.ex:142]:
```suggestion
    state = %{state | last_pts: max(sample.pts state.last_pts), last_dts: max(state.last_dts, sample.dts)}
```

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: lib/membrane_mp4/muxer/cmaf.ex (hunk 2 above)
Comment (varsill) [reply lib/membrane_mp4/muxer/cmaf.ex]: done

File: lib/membrane_mp4/muxer/cmaf.ex (hunk 1 above)
Comment (varsill) [reply lib/membrane_mp4/muxer/cmaf.ex]: done

**Review by mat-hek — APPROVED** [review]

**Pull request closed without merging**